org.mwc.debrief.legacy/src/Debrief/GUI/Views/CoreCoordinateRecorder.java

```diff
@@ -418,7 +418,7 @@ private void calculateScaleWidth(TrackData td, PlainProjection proj,
 
     td.setScaleWidth(tick_step);
     td.setScaleAmount(_scaleStep);
-    td.setScaleUnit(_DisplayUnits.getUnits());
+    td.setScaleUnit(scaleBarUnit);

```

Comment:
@IanMayo I think you are right. It is not needed in this case, and I think it is ok to remove it.

I have kept it because I wanted it to look similar to the original implementation in https://github.com/debrief/debrief/blob/develop/org.mwc.cmap.legacy/src/MWC/GUI/Chart/Painters/ScalePainter.java
Hmmm, and I think it is also correct to remove it from there. The idea would be move the object "_myUnits" to the ScalePainter. It is the "current unit" of the Scale Object. The unit converter shouldn't keep it.

Reply:
@saulhidalgoaular - let's delete the stored item then, in both cases.  I'm sure the compiler will pick up any problems, but you'll still need to visually check both Scales (Debrief Plot and PPT Slide) work properly.